web/src/components/DiffsTable.vue

```diff
@@ -36,6 +36,7 @@ export default class DiffsTable extends Vue {
   @Prop() loaded!: boolean;
   @Prop() diffs!: Diff[];
   @Prop({ default: "" }) search!: string;
+  itemsLoaded = false;
```

Comment:
I still don't really get why this is needed. There should only be one point upon which everything is loaded?

Reply:
I double checked and it is indeed not needed.